xwiki-platform-core/xwiki-platform-rendering/xwiki-platform-rendering-ui/src/main/resources/Rendering/Translations.xml

```diff
@@ -42,7 +42,7 @@
   <content>admin.syntaxes=Syntaxes
 admin.syntaxes.description=Choose the default page syntax and configure the available markup syntaxes.
 Rendering.RenderingConfigClass_disabledSyntaxes=Disabled Syntaxes
-Rendering.RenderingConfigClass_disabledSyntaxes.hint=The list of markup syntaxes that pages cannot be written in
+Rendering.RenderingConfigClass_disabledSyntaxes.hint=List of markup syntaxes in which pages cannot be written
```

Comment:
Please keep the `The X` format, for consistency with other locations, as mentioned in a previous comment.

Reply:
Done.